### ISSUE TITLE
Page Management needs a validation on page name

### ISSUE THREAD

**Issue by leigh-pointer** [oqtane/oqtane.framework#1300]
It should not be possible to add/save the page without a page name.

**Comment by sbwalker** [comment]
both the add and edit components already contain validation logic to ensure the name is provided before saving - have you been able to determine why its not working?

**Comment by leigh-pointer** [comment]
When save button is click and the page has no name I see and error and log is created where as I would expect to be warned with why the page was not created / saved.

**Comment by sbwalker** [comment]
Perhaps the problem in the add.razor component is that the _name is not initialized to string.empty to begin with?

**Comment by leigh-pointer** [comment]
@sbwalker yes that is it.  Just added the string.Empty and receive the correct warning.  Should l PR it ?

**Comment by sbwalker** [comment]
absolutely

**Comment by leigh-pointer** [comment]
Fixed in PR "Check now displays warning instead of an Error" #1302
